cli/src/main/java/org/jboss/sbomer/cli/feature/sbom/command/AbstractGenerateCommand.java

```diff
@@ -124,170 +125,182 @@ protected String toolVersion() {
 
     @Override
     public Integer call() {
-        // Make sure there is no context
-        MDCUtils.removeContext();
-        MDCUtils.addIdentifierContext(parent.getBuildId());
-
-        // Fetch build information
-        Build build = pncService.getBuild(parent.getBuildId());
-
-        if (build == null) {
-            throw new MissingPncBuildException("Could not fetch the PNC build with id '{}'", parent.getBuildId());
-        }
-
-        // Filter only valid PNC builds
-        if (!isValidBuild(build)) {
-            throw new InvalidPncBuildStateException(
-                    "Build '{}' is not valid! Build cannot be temporary and progress needs to be 'FINISHED' with status 'SUCCESS' or 'NO_REBUILD_REQUIRED'. Currently: temporary: {}, progress: '{}', status: '{}'",
-                    parent.getBuildId(),
-                    build.getTemporaryBuild(),
-                    build.getProgress(),
-                    build.getStatus());
-        }
+        try {
+            // Make sure there is no context
+            MDCUtils.removeContext();
+            MDCUtils.addIdentifierContext(parent.getBuildId());
+            MDCUtils.addOtelContext(OtelCLIUtils.getOtelContextFromEnvVariables());
 
-        // Filter only valid PNC build types
-        if (!isValidBuildType(build)) {
-            throw new UnsupportedPncBuildException(
-                    "The generation of SBOMs for the build type '{}' is not yet implemented!",
-                    build.getBuildConfigRevision().getBuildType());
-        }
+            OtelCLIUtils.startOtel("generate-cli");
```

Comment:
Wouldn't be better to make it explicit: `generate-pnc-build`?

Reply:
ok I updated the names just before you comment, let me know.